### DISCUSSION TITLE
3D Model displaying shadow but not model

### DISCUSSION THREAD

**Discussion by kelvenchin** [sceneview/sceneform-android#194] [Q&A]
![image](https://user-images.githubusercontent.com/92576222/139417016-27ff765f-3ac6-4671-b033-e17cf8ebb122.png)
I put in my self created 3D model in .glb format but it only display shadow but not the model. Any reason will cause this things happen ?

**Comment by ThomasGorisse** [comment]
Your model might be too big or too upped.
Scale it down from Blender or from node localScale.

**Comment by kelvenchin** [comment]
I found that if I put the 3D model scale small enough it can be show and stick at the node properly, but if I scale it become bigger it will stick to me whenever how I move it ? I'm using emulator virtual scene because of I don't have a AR supported device. But my project is I want to place a 3D building and walk through the building to see details inside. Is it because the virtual scene is too small, I need to change to the AR support device to make it ?

**Comment by RGregat** [comment]
If I'm correct filament treats 1 unit as 1 meter. So make sure to scale your model in Blender with this information in mind.

**Reply by kelvenchin** [reply]
Means all the scale adjust in the blender will be better right.
